### DISCUSSION TITLE
✨ Baserow

### DISCUSSION THREAD

**Discussion by JigSawFr** [runtipi/runtipi-appstore#2220] [Requests]
### Description

Create your own online database without technical experience. Our user-friendly no-code
tool gives you the powers of a developer without leaving your browser.

- A spreadsheet database hybrid combining ease of use and powerful data organization.
- Easily self-hosted with no storage restrictions or sign-up on https://baserow.io/ to
- get started immediately.
- Alternative to Airtable.
- Open-core with all non-premium and non-enterprise features under
- the [MIT License](https://choosealicense.com/licenses/mit/) allowing commercial and
- private use.
- Headless and API first.
- Uses popular frameworks and tools like [Django](https://www.djangoproject.com/),
- [Vue.js](https://vuejs.org/) and [PostgreSQL](https://www.postgresql.org/).

### Official Website

https://baserow.io/

### Documentation link

https://baserow.io/user-docs

### Last application release & date

1.22 (15/01/2024)

### Application license

MIT License

### Source code repository link

https://gitlab.com/baserow/baserow

### Docker image link

https://baserow.io/docs/installation%2Finstall-with-docker

### Please confirm the following

- [X] I confirm that I've put the app name in the title of the discussion
- [X] I confirm that the application is actively maintained.
- [X] I confirm that the docker image of the application is actively maintained.
- [X] I confirm that I have indeed checked that no similar request for this application already exists

### Gathering crowds

Hey folks!

Please upvote 👍 this discussion to show your interest in this request! 

Thanks ⛺

**Comment by JigSawFr** [comment]
Originally requested by @andrewbrereton

**Comment by bst1n** [comment]
Missing Baserow prevents me from using Runtipi.

**Comment by ddlaws0n** [comment]
Would love to see Baserow added as it's more mature than Nocodb currently and very API friendly.
